docs/dev/test-configuration-options.md

```diff
@@ -83,7 +88,7 @@ __Example__:
 "platformName": "macOS 10.13", "platformName": "iOS", "platformName": "Android"
 ```
 
-## Browser W3C Capabilities (Optional)
+## Browser W3C Capabilities – Optional
```

Comment:
@walkerlj0 @titusfortner Having Optional and Required in parens is our standard style right now. We could bring this up in the editorial sync this week.

Reply:
Yeah @kim, but having partentheses in the title broke the links-- in other words, I am unable to link in-doc to a title if it has parentheses, so I had to change this

Reply:
@walkerlj0 oh I see, thank you for the heads up!